### PR TITLE
Ref Guide - SD - Overview

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Reference Guide - Systems - System Details

Basically all the top level sections around System Details (they're more confusing than I expected!). 

A note on tabs and subtabs in this section:
Essentially, everything in the top line is a tab, and everything on the second line is a subtab:

image

So for the `Details` tab, there are subtabs of `Overview`, `Properties`, `Remote Command`, etc. These subtabs (except `Overview`) all have their own sections in the Reference Guide, at the same level as the other tabs. This PR renames `system-details/system-details.adoc` to be `system-details/sd-details.adoc` to match the naming schema of the other files. It also splits out the content about the Details > Overview subtab into its own file called `system-details/sd-overview.adoc` to avoid the nesting problems. 

**Note** the `systems/systems-overview.adoc` and the `systems/system-details/sd-overview.adoc` pages are at different levels in the doc, and refer to two distinct sections of the UI (Systems > Overview and System Details > Overview).

# Target branches

To what product version this document applies?

- [ ] Manager-4.0
- [ ] Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

# Links

Partial: https://github.com/SUSE/spacewalk/issues/11449

### CODE REVIEW

**Comment by Loquacity** [comment]
image

**Comment by keichwa** [comment]
Yes, al this canbe very confusing and often difficult to name and sort correctly. Often the text of the menu entry and the title of the tab do not match or do not match exactly:

menu:Systems[Overview] has the title "System Overview" (should be just "Overview" or "Systems Overview" [plural]).

At other places, we use "All" or "All Types" or something with "list" instead of "Overview".

**Comment by Loquacity** [comment]
> Yes, al this canbe very confusing and often difficult to name and sort correctly. Often the text of the menu entry and the title of the tab do not match or do not match exactly:
> 
> menu:Systems[Overview] has the title "System Overview" (should be just "Overview" or "Systems Overview" [plural]).
> 
> At other places, we use "All" or "All Types" or something with "list" instead of "Overview".

Yeah, I created https://github.com/SUSE/spacewalk/issues/12041 just now so we can go back and address these holistically once the chapter review is complete.

File: modules/reference/pages/systems/system-details/sd-overview.adoc [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+For more information about system locks, see xref:client-configuration:system-locking.adoc[].
+
+
+
+== Subscribed Channels
+
+This section shows the base and child software channels this client is subscribed to.
+
+Click btn:``Alter Channel Subscriptions`` to change the channel subscriptions for this client.
```
Comment (keichwa) [inline modules/reference/pages/systems/system-details/sd-overview.adoc:98]:
```suggestion
Click btn:[Alter Channel Subscriptions] to change the channel subscriptions for this client.
```
Maybe, your variant will also work.
From time to time I must say: I'm still not in love with adoc.

File: modules/reference/pages/systems/system-details/sd-overview.adoc [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+Click a channel name to go to the [guimenu]``Basic Channel Details`` section.
```
Comment (keichwa) [inline modules/reference/pages/systems/system-details/sd-overview.adoc:99]:
```suggestion
Click a channel name to go to the [guimenu]``Channel Details`` tab.
```
``Basic Channel Details`` is one section of the ``Channel Details`` page or tab.

File: modules/reference/pages/systems/system-details/sd-overview.adoc [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+| Registered
+| The time that the client registered with {productname}.
+
+| Last booted
+| The time that the client was most recently started or rebooted.
+
+|===
+
+
+Click btn:``Schedule system reboot`` to set a time for the client to reboot.
```
Comment (keichwa) [inline modules/reference/pages/systems/system-details/sd-overview.adoc:129]:
```suggestion
Click btn:[Schedule system reboot] to set a time for the client to reboot.
```

File: modules/reference/pages/systems/system-details/sd-overview.adoc [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+[[sdoverview-sysproperties-columns]]
+.System Properties Columns
+[cols="1,1", options="header"]
+|===
+
+| Column Name
+| Description
+
+| System types
+| Lists the system types and add-on types currently applied to the client.
```
Comment (keichwa) [inline modules/reference/pages/systems/system-details/sd-overview.adoc:148]: I'm not about add-on types here.  I think this is just [salt] or [management] (for "tradiitional")

File: modules/reference/pages/systems/systems-overview.adoc [hunk 5]
```diff
@@ -1,8 +1,23 @@
...
+The list contains information about the clients, including available software updates, the base channel they are subscribed to, and whether they are a traditional or Salt client.
+
+Click the name of a client to go to the [guimenu]``System Details Overview`` page.
+For more information about the [guimenu]``System Details Overview`` page in the {webui}, see xref:reference:systems/system-details/sd-details.adoc[].
+
+Check a client to add it to the system set manager.
+For more information about the system set manager, see xref:client-configuration:using-ssm.adoc[].
+
+Click btn:[Add Systems] to go to menu:Systems[Bootstrapping].
+For more about registering new clients, see xref:client-configuration:registration-overview.adoc[].
```
Comment (keichwa) [inline modules/reference/pages/systems/systems-overview.adoc:14]:
```suggestion
```
If wanted or needed, here we'd better just reference 
```
xref:reference:systems/bootstrapping.adoc[]
```

File: modules/reference/pages/systems/system-details/sd-overview.adoc (hunk 1 above)
Comment (Loquacity) [reply modules/reference/pages/systems/system-details/sd-overview.adoc:98]: No, that's my brain consistently not remembering the btn syntax. Thanks!

File: modules/reference/pages/systems/system-details/sd-overview.adoc (hunk 4 above)
Comment (Loquacity) [reply modules/reference/pages/systems/system-details/sd-overview.adoc:148]: I wasn't sure about that. Thanks!

**Pull request merged**